### DISCUSSION TITLE
We should deploy our client app on cloud platforms like vercel/netlify/aws for  quick demo

### DISCUSSION THREAD

**Discussion by Salmandabbakuti** [suhailkakar/Decentralized-YouTube#14] [Ideas]
Lets deploy our client app on cloud platforms like vercel/netlify/aws for a quick demo or it is deployed already?

**Comment by suhailkakar** [comment]
Hey Salman!

Sounds good!
But I am thinking about some of the users who might misuse the app :(

**Reply by suhailkakar** [reply]
I might not deploy the app until I implement a good solution for it 

P.S: I have a solution but it might take some time to implement :)
